openstudiocore/src/model/test/ScheduleInterval_GTest.cpp

```diff
@@ -242,3 +246,28 @@ TEST_F(ModelFixture, Schedule_VariableInterval3)
 
   EXPECT_TRUE(schedule.optionalCast<ScheduleVariableInterval>());
 }
+
+TEST_F(ModelFixture, Schedule_File)
+{
+  Model model;
+
+  path p = resourcesPath() / toPath("model/schedulefile.csv");
+
+  ExternalFile externalfile(model, openstudio::toString(p));
+  EXPECT_EQ(openstudio::toString(p), externalfile.fileName());
```

Comment:
I think this should equal "schedulefile.csv" and we should also test that the file was copied.